remote-api/src/main/java/org/alfresco/repo/web/scripts/content/ContentGet.java

```diff
@@ -65,6 +68,16 @@ public class ContentGet extends StreamContent implements ServletContextAware
     private NamespaceService namespaceService;
     private ContentService contentService;
 
+    private Set<String> nonAttachContentTypes = Collections.emptySet();
+
+    public void setNonAttachContentTypes(String nonAttachAllowListStr)
+    {
+        if ((nonAttachAllowListStr != null) && (! nonAttachAllowListStr.isEmpty()))
```

Comment:
There is an extra space between `!` and `nonAttachAllowListStr`.
Either way, there is a much cleaner way to inject comma-separated Strings as Lists by leveraging Spring features.
Look at https://github.com/Alfresco/alfresco-enterprise-repo/pull/819/files#diff-be731d09427ccf7e6a2346777d6fdc801b5ef77d5c9459476ded44d396fa9fdbR63 for example.
It would be good to use the latter.